java/ql/src/experimental/semmle/code/java/Logging.qll

```diff
@@ -9,17 +9,22 @@ class LoggingCall extends MethodAccess {
   LoggingCall() {
     exists(RefType t, Method m |
       t.hasQualifiedName("org.apache.log4j", "Category") or // Log4j 1
-      t.hasQualifiedName("org.apache.logging.log4j", ["Logger", "LogBuilder"]) or // Log4j 2      
+      t.hasQualifiedName("org.apache.logging.log4j", ["Logger", "LogBuilder"]) or // Log4j 2
       t.hasQualifiedName("org.apache.commons.logging", "Log") or
       // JBoss Logging (`org.jboss.logging.Logger` in some implementations like JBoss Application Server 4.0.4 did not implement `BasicLogger`)
       t.hasQualifiedName("org.jboss.logging", ["BasicLogger", "Logger"]) or
       t.hasQualifiedName("org.slf4j.spi", "LoggingEventBuilder") or
       t.hasQualifiedName("org.slf4j", "Logger") or
       t.hasQualifiedName("org.scijava.log", "Logger") or
+      t.hasQualifiedName("com.google.common.flogger", "LoggingApi") or
       t.hasQualifiedName("java.lang", "System$Logger") or
-      t.hasQualifiedName("java.util.logging","Logger")
+      t.hasQualifiedName("java.util.logging", "Logger") or
+      t.hasQualifiedName("android.util.Log", _)
     |
-      m.getDeclaringType().(RefType).extendsOrImplements*(t) and
+      (
+        m.getDeclaringType().getASourceSupertype*() = t or
+        m.getDeclaringType().(RefType).extendsOrImplements*(t)
```

Comment:
Not sure if this is still needed. But even if you keep it, the cast to `RefType` is not needed because `getDeclaringType` has `RefType` as result type.

Reply:
@Marcono1234 Changes done!